apps/src/code-studio/components/SoundCategory.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import * as color from "../../util/color";
+
+/**
+ * A component for displaying a sound category.
+ */
+var SoundCategory = React.createClass({
+  propTypes: {
+    displayName: React.PropTypes.string,
+    category: React.PropTypes.string,
+    onSelect: React.PropTypes.func.isRequired
+  },
+
+  selectCategory: function () {
+    this.props.onSelect(this.props.category);
+  },
+
+  render: function () {
+    var styles = {
```

Comment:
I've seen multiple locations for styles in our code base. Is there a preferred way?